src/Microsoft.Azure.WebJobs.ServiceBus/IMessagingProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.Azure.ServiceBus.Core;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    public interface IMessagingProvider
```

Comment:
Why the new interface?

Reply:
I honestly don't remember... I was going nuts moving everything to DI and probably just went a bit too far :-)

Should be able to remove this and go back to using `MessagingProvider` everywhere my next commit.